### PR TITLE
Edit pcr test series

### LINKED ISSUE #65: PCR Test for everyone
Each attendee needs a PCR Test which are pooled. So each attendee needs to be assigned to a pool so in the end, if a pool is positive it is knows who was in the pool.

This needs to be done multiple times, currently planned 2 times.

### PR DESCRIPTION
**Pull request by KordonDev**
closes #65

### CODE REVIEW

File: frontend/src/components/DateAndTimeWithDate.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  get dateOnly(): string {
+    return dateIsoString(this.date);
+  }
+
+  get timeOnly(): string {
+    return timeIsoString(this.date);
+  }
+
+  private dateChangedInteral(newDate: string) {
+    dateAndTimeAsIsoString(newDate, this.timeOnly)
```
Comment (crymis) [inline frontend/src/components/DateAndTimeWithDate.vue:44]: It seems that this row does not do anything.

File: frontend/src/services/pcrTestSeries.ts [hunk 2]
```diff
@@ -1,47 +1,68 @@
...
-  return deleteData(`pcr-test-series/${id}`, withAuthenticationHeader());
-};
+  return deleteData(`pcr-test-series/${id}`, withAuthenticationHeader())
+}
+
+const converDates = (pcrTestSeries: PcrTestSeriesResponse): PcrTestSeries => {
+  return {
+    ...pcrTestSeries,
+    start: new Date(pcrTestSeries.start),
+    end: new Date(pcrTestSeries.end),
```
Comment (crymis) [inline frontend/src/services/pcrTestSeries.ts:66]: convertDates, you missed a 't' 🫖

Comment (KordonDev) [reply frontend/src/services/pcrTestSeries.ts:66]: done

**Comment by KordonDev** [comment]
All timezones are bad

**Pull request merged**